src/Fist/Http/tests/RequestTest.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace FistTest;
+
+use Fist\Http\Request;
+use Fist\Testing\TestCase;
+use Fist\Testing\WithSuperGlobals;
+
+/**
+ * @SuppressWarnings(PHPMD.ExcessivePublicCount)
+ */
+class RequestTest extends TestCase
```

Comment:
__[phpmd]__
* The class RequestTest has 16 public methods. Consider refactoring RequestTest to keep number of public methods under 10.

<!-- Comment by [SideCI](https://sideci.com) -->